client/webserver/site/dist/entry.js

```diff
@@ -1,2 +1,2 @@
alue:function(e){if(this.haveAssetOrders(e))return!0;for(var t=0,n=Object.values(this.user.exchanges);t<n.length;t++){var r=n[t];if(r.pendingFee&&r.pendingFee.assetID===e)return!0}return!1}},{key:"order",value:function(e){for(var t=0,n=Object.values(this.user.exchanges);t<n.length;t++)for(var r=n[t],a=0,o=Object.values(r.markets);a<o.length;a++){var i=o[a];if(i.orders){var s,c=Ir(i.orders);try{for(c.s();!(s=c.n()).done;){var u=s.value;if(u.id===e)return u}}catch(e){c.e(e)}finally{c.f()}}}return null}},{key:"canAccelerateOrder",value:function(e){var t;t=e.sell?e.baseID:e.quoteID;var n=this.walletMap[t];if(!(n&&16&n.traits))return!1;if(e.matches)for(var r=0;r<e.matches.length;r++){var a=e.matches[r];if(a.swap&&a.swap.confs&&0===a.swap.confs.count)return!0}return!1}},{key:"unitInfo",value:function(e,t){var n=this.assets[e];if(n)return n.info.unitinfo;if(!t)throw Error("no supported asset info for id = ".concat(e,", and no exchange info provided"));return t.assets[e].unitInfo}},{key:"conventionalRate",value:function(e,t,n){var r=[this.unitInfo(e),this.unitInfo(t)],a=r[1],o=r[0].conventional.conversionFactor/a.conventional.conversionFactor;return n/ft*o}},{key:"walletDefinition",value:function(e,t){var n=this.assets[e].info;return""===t?n.availablewallets[n.emptyidx]:n.availablewallets.filter((function(e){return e.type===t}))[0]}},{key:"currentWalletDefinition",value:function(e){return this.walletDefinition(e,this.assets[e].wallet.type)}},{key:"fetchBalance",value:(n=o(y().mark((function e(t){var n;return y().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return e.next=2,nt("/api/balance",{assetID:t});case 2:if(n=e.sent,this.checkResponse(n)){e.next=5;break}throw new Error("failed to fetch balance for asset ID ".concat(t));case 5:return e.abrupt("return",n.balance);case 6:case"end":return e.stop()}}),e,this)}))),function(e){return n.apply(this,arguments)})},{key:"checkResponse",value:function(e,t){return!(!e.requestSuccessful||!e.ok)||(this.user.inited&&!t&&this.notify(Xt(ke(ae),e.msg,5)),!1)}},{key:"signOut",value:(t=o(y().mark((function e(){var t;return y().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return e.next=2,nt("/api/logout");case 2:if(t=e.sent,this.checkResponse(t)){e.next=6;break}return De.hide(this.page.profileBox),e.abrupt("return");case 6:Ve.clearAllStore(),Ve.removeAuthCK(),window.location.href="/login";case 9:case"end":return e.stop()}}),e,this)}))),function(){return t.apply(this,arguments)})}]),e}(),Nr=(u(Fr={},3,"good"),u(Fr,5,"bad"),u(Fr,4,"warn"),Fr);function Ur(e){return e.replace(/^\//,"").split("/")[0].split("?")[0].split("#")[0]}function zr(){var e=new Date,t=e.getHours().toString().padStart(2,"0"),n=e.getMinutes().toString().padStart(2,"0"),r=e.getSeconds().toString().padStart(2,"0"),a=e.getMilliseconds().toString().padStart(3,"0");return"".concat(t,":").concat(n,":").concat(r,".").concat(a)}function jr(e,t){e.classList.remove("bad","warn","good"),e.classList.add(Nr[t])}var _r=new Mr;Je=_r,_r.start()})()})();
```

Comment:
Only change I see in the bundle is `this.commitHash="fb5d9ffd9fc16487f659f6dc99e32e43c0aa2711"`, so yeah, the npm fix was not a runtime vuln.  Probably not even a vuln for dev, but done anyway.

Reply:
New bundle still working fine for me.
Have had this branch on mainnet all day, just without the last three commits, until now.